woocommerce/woocommerce-gateway-stripe
# Hide Multibanco on the checkout page when the Stripe account's country doesn't support it

Fixes #3227 

This PR fixes the 

## Changes proposed in this Pull Request:
- Added the supported country list for Multibanco. Find the list of supported countries in Stripe documentation [here](https://docs.stripe.com/payments/multibanco).
- Multibanco is available on the settings page only when the connected Stripe account is of a supported country.

## Testing instructions

1. Create a Stripe account associated with a country that's not in [this list](https://docs.stripe.com/payments/multibanco#:~:text=Stripe%20users%20in%20the%20following%20countries%20can%20accept%20Multibanco%20payments), like India.
2. Connect your store to this account.
3. Select Euro as your store currency.
4. Go to the Stripe settings page and confirm that Multibanco is not in the payment method list. 
5. Now connect to a US Stripe account and recheck the payment methods list.
6. This time Multibanco should be present in the payment methods list.
7. Enable Multibanco.
8. As a shopper make a purchase with Multibanco. Confirm that the payment is successful and the order is placed.
